src/guiapi/include/Jogwheel.hpp

```diff
@@ -38,6 +38,19 @@ class Jogwheel {
         Held
     };
 
+    //structure to be read in rtos thread (outside interrupt)
+    //size must be 32 bit to be atomic
```

Comment:
I'd recommend adding static_assert(sizeof(encoder_t)==4) somewhere when you rely on the exact size